targets/PythonSdk/make.js

```diff
@@ -0,0 +1,407 @@
+
+    for(var i=0; i< apis.length; i++)
+        makeApi(apis[i], sourceDir, apiOutputDir);
+
+    generateSimpleFiles(apis, sourceDir, apiOutputDir);
+}
+
+// generate.js looks for some specific exported functions in make.js, like:
+exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
+    var locals = {
+        apis: apis,
+        buildIdentifier: exports.buildIdentifier,
+        errorList: apis[0].errorList,
+        errors: apis[0].errors,
+        friendlyName: "PlayFab Python Combined Sdk",
+        sdkVersion: exports.sdkVersion
+    };
+
+    // Builds the client api.  The provided "api" variable is a single object, the API_SPECS/client.api.json as an object
+    console.log("Generating Combined Client/Server api from: " + sourceDir + " to: " + apiOutputDir);
+
+    templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+    for (var i = 0; i < apis.length; i++)
+        makeApi(apis[i], sourceDir, apiOutputDir);
+    generateSimpleFiles(apis, sourceDir, apiOutputDir);
+}
+
+// Unlike source, Templates are written one file at a time.
+// You may want to write a helper function to write each template file, so you can call it from multiple places
+function MakeExampleTemplateFile(sourceDir, apiOutputDir) {
+    // Each template must be given any variables/information that it needs for generation.
+    // This might include apis, datatypes, custom functions defined in this make.js file, or anything else you want
+    var locals = {};
+
+    locals.GeneratedText = "This is generated text"; // A specific variable we wish to access in exampleTemplate.txt.ejs
+    locals.sdkVersion = exports.sdkVersion; // exports.sdkVersion is automatically injected into this file from generate.js, and comes from SdkManualNotes.json - you must provide your target in that file
+
+    // Compiles the source .ejs file into a template function.
+    var template = getCompiledTemplate(path.resolve(sourceDir, "templates/exampleTemplate.txt.ejs"));
+
+    // Call the template function, which executes the template, and evaluates all the ejs tags/logic
+    var generatedTemplateText = template(locals);
+
+    // generatedTemplateText is an in-memory string of the output file.  At this point, you just write it to the destination:
+    writeFile(path.resolve(apiOutputDir, "exampleTemplate.txt"), generatedTemplateText);
+}
+
+function makeDataTypes(apis, sourceDir, apiOutputDir) {
+    var modelTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Model.py.ejs"));
+    var modelsTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Models.py.ejs"));
+    var enumTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/Enum.py.ejs"));
+    var jsonTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/PlayFabJson.py.ejs"));
+
+    var makeDatatype = function (datatype, api) {
+        var modelLocals = {
```

Comment:
Go through this list and make sure you're using all of these vars/functions in these templates.